src/stories/components/Path.stories.tsx

```diff
@@ -32,49 +27,29 @@ const sampleTasks: { [key: string]: Task } = {
   'In Progress': sampleTask5,
 };
 
-
-
-const epicTemplate: Story<ComponentProps<typeof EpicStatusPath>> = (args) => (
-  <EpicStatusPath {...args} />
-);
-
-export const EpicPath = epicTemplate.bind({});
-
-EpicPath.args = {
-  status: 'Review',
-};
-EpicPath.argTypes = {
-  status: {
-    defaultValue: Object.keys(EPIC_STATUSES),
-    control: {
-      type: 'select',
-      options: Object.keys(EPIC_STATUSES),
-    },
-  },
+type Props = ComponentProps<typeof Path>;
+interface StoryProps extends Omit<Props, 'task'> {
+  task: Task;
+}
+const TaskTemplate = ({ task, ...rest }: StoryProps) => {
+  return (
+    <TaskStatusPath
+      task={Object.keys(sampleTasks).map((t) => sampleTasks[t])}
+      {...rest}
```

Comment:
@jgerigmeyer Kinda stumped on making the select control work from [this push](https://github.com/oddbird/Metecho/pull/859/commits/45dd2e9af66abdec644588852ef3d8019801ddf3).  Looks like the 'task' prop we are passing is returning an array of Tasks (`Tasks[]`) and I'm not real clear on how to connect the selected task from the control to the component's prop.

Reply:
☑ Fixed in https://github.com/oddbird/Metecho/commit/0622f6d029c488768bc003345e5a62373b88de6a